# Grasp approch vector in camera coordinates
geometry_msgs/Vector3 approach
# Grasp score
float64 score
# Contact point in camera coordinates
geometry_msgs/Point32 position
# Instance id
float64 obj_id
